python/paddle/__init__.py

```diff
@@ -726,6 +727,7 @@
     'zeros_like',
     'maximum',
     'topk',
+    'top_p_sampling',
```

Comment:
If ·top_p_sampling· is a non-public API, just leave the import to be called and do not add it to list of \_\_all\_\_